app/controllers/api/base_controller/action.rb

```diff
@@ -35,12 +35,17 @@ def create_ems_resource(type, data, supports: false)
           klass = ems.class_by_ems(base_klass)
           raise "#{type} not supported by Provider #{ems.name}" unless klass
 
-          ensure_supports(type, klass, :create) if supports
+          ensure_supports(type, klass, supports) if supports
 
-          full_action_results(yield(ems, klass)) { "Creating #{type.to_s.singularize.titleize} #{model_name} for Provider #{ems.name}" }
+          full_action_results(yield(ems, klass)) { "#{action} #{type.to_s.singularize.titleize} #{model_name} for Provider #{ems.name}" }
```

Comment:
Bug?  Should this be action_phrase? Seems rubocop caught this too...maybe this needs a spec?